scripts/stress-test.pl

```diff
@@ -0,0 +1,24 @@
+use constant BYTES_IN_MB => 1024 * 1024;
```

Comment:
It's in the description. Basically, so I can use the ubuntu:14.04 image for the tests, and not download some other much larger image.